config/config.go

```diff
@@ -165,6 +167,32 @@ func (cfg *AuctionTimeouts) validate(errs []error) []error {
 	return errs
 }
 
+type Compression struct {
+	Request  CompressionConfig `mapstructure:"request"`
+	Response CompressionConfig `mapstructure:"response"`
+}
```

Comment:
We need to make it clear that this compression config is for Prebid Server itself and not for bid adapters. That is, these settings do not apply to bidder requests. For bidder requests, gzip decoding is supported without a config option and encoding is configured per adapter yaml file.

Reply:
How about we change the names to be `pbs_request` and `pbs_response` to make it more clear that this is only for prebid-server requests and responses? So in the final config, these will be read like `compression.pbs_request.enabled` and `compression.pbs_response.enabled`

Reply:
Naming is hard. I like adding a qualifier to the name.

PBS is also the default prefix for environment variables, so using a `pbs_` prefix for would look like `PBS_COMPRESSION_PBS_RESPONSE_ENABLED`.

If we change the name of the higher level `compression` object, then we restrict ourselves from using that for any future global bidder compression or cookie compression settings.

¯\_(ツ)_/¯

Reply:
This was one of the reasons I had kept this to just `compression.request.enabled` and `compression.response.enabled` so when using environment variables to set the config, with the `PBS_` prefix they will be read as `PBS_COMPRESSION_REQUEST_ENABLED` and `PBS_COMPRESSION_RESPONSE_ENABLED`. IMO that should be clear that this is only meant for PBS requests and responses. I am happy to add comments in the code to clarify that since we don't really have any other documentation for config options.